File: suco/suco/src/main/java/com/example/suco/dto/sos/payment/hoadon/response/ThanhToanResponseDTO.java
package com.example.suco.dto.sos.payment.hoadon.response;

import java.math.BigDecimal;

public class ThanhToanResponseDTO {

    private Long hoaDonId;

    private String trangThai;

    private BigDecimal tongThanhToan;

    private String message;

    public Long getHoaDonId() {
        return hoaDonId;
    }

    public void setHoaDonId(Long hoaDonId) {
        this.hoaDonId = hoaDonId;
    }

    public String getTrangThai() {
        return trangThai;
    }

    public void setTrangThai(String trangThai) {
        this.trangThai = trangThai;
    }

    public BigDecimal getTongThanhToan() {
        return tongThanhToan;
    }

    public void setTongThanhToan(BigDecimal tongThanhToan) {
        this.tongThanhToan = tongThanhToan;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: CanhBao/app/src/main/java/com/example/canhbao/data/model/hoadon/payment/ThanhToanResponseDTO.kt
package com.example.canhbao.data.model.hoadon.payment

import java.math.BigDecimal
import java.time.LocalDateTime

data class ThanhToanResponseDTO (
    val thanhToanId: Long? = null,

    val hoaDonId: Long? = null,

    val trusoId: Long? = null,

    val phuongThucThanhToan: String? = null,

    val maGiaoDich: String? = null,

    val trangThai: String? = null,

    val thanhTien: BigDecimal? = null,

    val soTienGiam: BigDecimal? = null,

    val tongThanhToan: BigDecimal? = null,

    val message: String? = null,

    val createdAt: String? = null
)